script/firebaseScriptLogWork.js

```diff
@@ -54,6 +54,61 @@ var database = firebase.database();
 var ui = new firebaseui.auth.AuthUI(firebase.auth());
 
 var uiConfig = {
+callbacks: {
+signInSuccessWithAuthResult: function(authResult, redirectUrl) {
+// User successfully signed in.
+// Return type determines whether we continue the redirect automatically
+// or whether we leave that to developer to handle.
+return true;
+},
+uiShown: function() {
+// The widget is rendered.
+// Hide the loader.
+document.getElementById('loader').style.display = 'none';
+}
+},
+// Will use popup for IDP Providers sign-in flow instead of the default, redirect.
+signInFlow: 'popup',
+signInSuccessUrl: 'workLoggingSystem.html',
+signInOptions: [
+// Leave the lines as is for the providers you want to offer your users.
+{
+provider: firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+scopes: [
+        'https://www.googleapis.com/auth/contacts.readonly'
+    ],
+},
+firebase.auth.EmailAuthProvider.PROVIDER_ID
+]
+// Terms of service url.
+// tosUrl: '<your-tos-url>',
+// // Privacy policy url.
+// privacyPolicyUrl: '<your-privacy-policy-url>'
+};  
+
+function createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category,creationDate) {
+
+  var taskIdNumber = getIdNumber();
+  var taskId = category[0]+taskIdNumber;
+  var loggedWorkTotalTime = 0;
+  var workDone = 0;
+
+  database.ref('Tasks/' + taskId).set({
+    Title: title,
+    Description: des,
+    Priority: priority,
+    Difficulty: difficulty,
+    Creator: creator,
+    Assignee: assignee,
+    ET: estimatedTime,
+    Status: status,
+    Category: category,
+    LogWorkTotalTime: loggedWorkTotalTime,
+    CreationDate: creationDate,
+    WorkDone: workDone
+  });
+
+  newTaskCreatedSuccessfully();
```

Comment:
Repeated implementation, Severity High -> the whole code is getting burst due to this PR. Please take action immediately